src/pages/Lend/LoanChannelCategoryRecommendedByLenders.vue

```diff
@@ -0,0 +1,517 @@
+<template>
+	<div>
+		<div class="row">
+			<div class="small-12 columns heading-region">
+				<view-toggle browse-url="/lend-by-category" :filter-url="filterUrl" />
+				<p class="tw-text-small">
+					<router-link to="/lend-by-category">
+						All Loans
+					</router-link> >
+					<span class="show-for-large">{{ loanChannelName }}</span>
+				</p>
+				<h1 class="tw-mb-2">
+					{{ loanChannelName }}
+				</h1>
+				<p v-if="loanChannelDescription"
+					class="page-subhead show-for-large tw-mb-4"
+				>
+					{{ loanChannelDescription }}
+				</p>
+				<p v-else>
+					We couldn't find any loans for this search.
+					<router-link to="/lend-by-category">
+						Browse these loans
+					</router-link>.
+				</p>
+			</div>
+		</div>
+
+		<div class="row">
+			<div class="columns small-12" v-if="loans.length > 0">
+				<div class="loan-card-group row small-up-1 large-up-2 xxlarge-up-3">
+					<loan-card-controller
+						v-for="loan in loans"
+						:items-in-basket="itemsInBasket"
+						:is-visitor="isVisitor"
+						:key="loan.id"
+						:loan="loan"
+						loan-card-type="GridLoanCard"
+					/>
+				</div>
+				<kv-pagination v-if="totalCount > 0" :total="totalCount" :limit="limit" @page-change="pageChange" />
+				<div v-if="totalCount > 0" class="loan-count tw-text-tertiary">
+					{{ totalCount }} loans
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+/* This component is for the recommended loans by lenders category page.
+* It is very similar to src/pages/Lend/LoanChannelCategoryControl.vue with
+* extraneous functionality removed. The other differences being in the query
+* this component uses and this component also filters out loans that do not
+* have monthly repayment interval.
+* TODO
```

Comment:
Is there a follow-up ticket for pagination? I noticed it didn't align in my functional testing as well.